webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -37,10 +37,13 @@ class ManageManifestModal extends Component {
       this.props.loadOrganization();
       this.props.loadManifestHistory();
     }
+
+    if (!prevProps.manifestActionStarted && this.props.manifestActionStarted) {
+      this.hideDeleteManifestModal();
+    }
   }
 
   hideModal = () => {
-    if (this.props.deleteManifestModalIsOpen) this.hideDeleteManifestModal();
```

Comment:
Was going to remind you also to remove `deleteManifestModalIsOpen` from `index.js` `mapStateToProps` but it appears it's already gone